config/initializers/stripe.rb

```diff
@@ -12,6 +12,7 @@ class << self
 
 Rails.application.reloader.to_prepare do
   Stripe.api_key = ENV['STRIPE_INSTANCE_SECRET_KEY']
+  Stripe.api_key = ENV['STRIPE_SECRET_TEST_API_KEY'] if Rails.env.test? || ENV["CI"]
```

Comment:
I personally would prefer this written as below :

```suggestion
  if Rails.env.test? || ENV["CI"]
    Stripe.api_key =  ENV['STRIPE_SECRET_TEST_API_KEY'] 
  else 
    Stripe.api_key =  ENV['STRIPE_INSTANCE_SECRET_KEY']
  end
```

Reply:
I'll take that suggestion ;-)

Reply:
Rubocop autorrected it in a slightly different way, hope that's ok too:
```

  Stripe.api_key = if Rails.env.test? || ENV["CI"]
                     ENV.fetch('STRIPE_SECRET_TEST_API_KEY', nil)
                   else
                     ENV.fetch('STRIPE_INSTANCE_SECRET_KEY', nil)
                   end
```